openwisp_radius/base/models.py

```diff
@@ -149,8 +149,8 @@
     'allowed to register via the user registration API.'
 )
 _GET_OPTIONAL_FIELDS_HELP_TEXT = _(
-    'Chose if this user field should be disabled, allowed or mandatory '
-    'during user registration.'
+    'Whether this field should be disabled, allowed or mandatory '
+    'in the user registration API.'
```

Comment:
This changes the help text on the model. so we need to run `./tests/manage.py makemigrations` for the migration to be updated.

Reply:
updated